recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-h3ulcb.cfg

```diff
@@ -44,6 +44,7 @@ vif = [ 'backend=DomD,bridge=xenbr0,mac=08:00:27:ff:cb:ce' ]
 
 vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1920x1080' ]
 
-vkb = [ 'backend=DomD,backend-type=linux,multi-touch-width=1024,multi-touch-height=768,multi-touch-num-contacts=10,unique-id=T:1000,feature-disable-pointer=1,feature-disable-keyboard=1' ]
+# multi-touch-width/height should be equal to physical touchscreen parameters when displ_be runs on top of DRM in DomD
```

Comment:
in both cases (WEston/DRM) multi-touch-num-contacts shall also be set properly to match HW